### PR TITLE
flux 0.127.1

### PR DESCRIPTION
**Pull request by branchv**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/flux.rb [hunk 1]
```diff
@@ -28,16 +28,30 @@ class Flux < Formula
     depends_on "pkg-config" => :build
   end
 
+  # NOTE: The version here is specified in the go.mod of influxdb.
+  # If you're upgrading to a newer influxdb version, check to see if this needs upgraded too.
```
Comment (carlocab) [inline Formula/flux.rb:32]: influxdb?

Comment (SMillerDev) [reply Formula/flux.rb:32]: This one is used in all Influxdata formulae, I think it should just be a formula at this point

Comment (chenrui333) [reply Formula/flux.rb:32]: could it be `influxdb-pkg-config`?

Comment (SMillerDev) [reply Formula/flux.rb:32]: I'd say `pkg-config-go-wrapper` or maybe just `pkg-config-wrapper`

**Review by chenrui333 — APPROVED** [review]

File: Formula/flux.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/flux.rb:32]: #84625

**Pull request closed without merging**